### PR TITLE
Add Stuttgarter Zeitung 

### PR DESCRIPTION
**Pull request by CWollinger**
Add support for:
https://www.stuttgarter-zeitung.de

### CODE REVIEW

File: app/scripts/blocker_stuttgarter.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { Blocker } from "./blocker_general";
+
+let blocker = new Blocker([
+
+    {
+        selector: 'article',
+        type: 'big'
+    },
+    {
+        selector: 'article mod-teaser',
```
Comment (dieparteidiepartei) [inline app/scripts/blocker_stuttgarter.js]: this is not a valid css selector in this context

File: app/scripts/blocker_stuttgarter.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        type: 'big'
+    },
+    {
+        selector: '.box-head',
+        type: 'big'
+    }
+]);
```
Comment (dieparteidiepartei) [inline app/scripts/blocker_stuttgarter.js]: what about `.mod-top-list` for example?

Comment (CWollinger) [reply app/scripts/blocker_stuttgarter.js]: added

**Comment by dieparteidiepartei** [comment]
Hi @CWollinger,
I'm having issues with the blocker for Stuttgarter Zeitung. Some Teasers are not blocked. The Selector should apply. Maybe you have an idea?
Cheers,
Fabian

**Pull request merged**